docs/source/executing-operations/subscription-support.mdx

```diff
@@ -1,120 +1,205 @@
 ---
 title: GraphQL subscriptions in the Apollo Router
```

Comment:
This article received helpful edit passes already and it hasn't changed much since, so it's less vital